sdk/kiosk/src/types/env.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { SuiAddress } from '@mysten/sui.js';
+
+/* A list of environments. */
+export type Environment = 'mainnet' | 'testnet' | 'devnet' | 'custom';
```

Comment:
I would suggest following the pattern we have in sui.js where we provide a variable for each environment (localConnection, testnetConnection etc)